diff -uprN msgpack-cli-0.7.0.old/src/MsgPack.Net45/MsgPack.Net45.csproj msgpack-cli-0.7.0/src/MsgPack.Net45/MsgPack.Net45.csproj
--- msgpack-cli-0.7.0.old/src/MsgPack.Net45/MsgPack.Net45.csproj	2016-07-03 16:48:42.000000000 +0300
+++ msgpack-cli-0.7.0/src/MsgPack.Net45/MsgPack.Net45.csproj	2016-07-09 09:36:21.498976828 +0300
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>MsgPack</RootNamespace>
     <AssemblyName>MsgPack</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
@@ -25,11 +25,11 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\..\bin\net452\</OutputPath>
+    <OutputPath>..\..\bin\net45\</OutputPath>
     <DefineConstants>TRACE;FEATURE_TAP;NETFX_45</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <DocumentationFile>..\..\bin\net452\MsgPack.XML</DocumentationFile>
+    <DocumentationFile>..\..\bin\net45\MsgPack.XML</DocumentationFile>
   </PropertyGroup>
   <PropertyGroup>
     <SignAssembly>true</SignAssembly>
@@ -1018,4 +1018,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
\ No newline at end of file
+</Project>
